src/controllers/friends.ts

```diff
@@ -8,113 +8,93 @@ const addFriend = (req: Request, res: Response) => {
   res.json("Hello World");
 };
 
-const sendFriendRequest = (req: Request, res: Response, next: NextFunction) => {
+const sendFriendRequest = async (req: Request, res: Response, next: NextFunction) => {
   const { user1Email, user2Email } = req.body;
 
-  const db = new DB();
-
-  db.createFriendRequest(user1Email, user2Email)
-    .then((value) => {
-      console.log(value);
```

Comment:
Remove console log

Reply:
This one is showing from the removed version, i already removed it